OpenRA.Mods.Common/Terrain/DefaultTerrain.cs

```diff
@@ -75,6 +75,7 @@ public class DefaultTerrain : ITemplatedTerrainInfo, ITerrainInfoNotifyMapCreate
 		public readonly bool EnableDepth = false;
 		public readonly float MinHeightColorBrightness = 1.0f;
 		public readonly float MaxHeightColorBrightness = 1.0f;
+		public readonly string DefaultPalette;
```

Comment:
Can we initialize this with `TileSet.TerrainPaletteInternalName;` and drop the yaml overrides?

Can we please also rename this to `Palette` for consistency with how we handle radar colour overrides?